crates/storage/libmdbx-rs/src/environment.rs

```diff
@@ -767,6 +773,16 @@ impl EnvironmentBuilder {
         self
     }
 
+    pub fn set_sync_bytes(&mut self, v: usize) -> &mut Self {
+        self.sync_bytes = Some(v as u64);
+        self
+    }
+
+    pub fn set_sync_period(&mut self, v: u64) -> &mut Self {
+        self.sync_period = Some(v);
+        self
+    }
+
```

Comment:
can you please add the doc comments for these options?

Reply:
Done. 
I can add a more verbose description from the "set" functions documentation if you like:
https://erthink.github.io/libmdbx/group__c__settings.html#ga7e64d98f7a8c3bdf7727627782f1be0a:~:text=%E2%97%86-,mdbx_env_set_syncbytes,-()
https://erthink.github.io/libmdbx/group__c__settings.html#ga7e64d98f7a8c3bdf7727627782f1be0a:~:text=%E2%97%86-,mdbx_env_set_syncperiod,-()